src/docs/release/breaking-changes/1-22-deprecations.md

```diff
@@ -0,0 +1,772 @@
+---
+title: Deprecated API removed after v1.22
+description: After reaching end of life, the following deprecated APIs were removed from Flutter.
+---
+
+## Summary
+
+In accordance with Flutter's [Deprecation Policy][], deprecated APIs that reached end of life after the
+1.22 stable release have been removed. This is the first time deprecated APIs have been removed from
+Flutter, and some of these deprecations predate our migration guide policy.
+
+All affected APIs have been compiled into this primary source to aid in migration. A
+[quick reference sheet][] is available as well.
+
+A [design document][] and [article][] are available for more context on Flutter's deprecation policy.
+
+[Deprecation Policy]: {{site.github}}/flutter/flutter/wiki/Tree-hygiene#deprecation
+[quick reference sheet]: /go/deprecations-removed-after-1-22
+[design document]: /go/flutter-deprecation-lifetime
```

Comment:
Kate, this link doesn't work and has broken the build. Please fix...

Reply:
```suggestion
[design document]: /go/deprecation-lifetime
```